test/conftest.py

```diff
@@ -524,9 +543,10 @@ def ensure_ids_are_correct_uuids(docs: list, document_store: object) -> None:
             d["id"] = str(uuid.uuid4())
 
 
-@pytest.fixture(params=["elasticsearch", "faiss", "memory", "milvus", "weaviate"])
+@pytest.fixture(params=["elasticsearch", "faiss", "memory", "milvus", "milvus2", "weaviate"])
 def document_store_with_docs(request, test_docs_xs, tmp_path):
     embedding_dim = request.node.get_closest_marker("embedding_dim", pytest.mark.embedding_dim(768))
+    print("oy!")
```

Comment:
This should probably not be merged to master :smile:  There are a few in the tests, please clean them up :wink:

Reply:
Oh, that shouldn't be there 😄. Now all debugging statements should be gone.